src/ui/sass/modules/_Results.scss

```diff
@@ -165,7 +167,8 @@ $results-cards-margin: calc(var(--yxt-base-spacing) / 2) !default;
     text-decoration: none;
     display: flex;
     align-items: center;
-    &:hover, &:focus 
+    &:hover .yxt-Results-viewAllLabel,
+    &:focus .yxt-Results-viewAllLabel
     {
       color: $color-brand-hover;
```

Comment:
this seems to change the text color too, is that what we want?

Reply:
by removing this, are we affecting legacy sites relying on this `&:hover` and `&:focus` (without viewAllLabel)?

Reply:
there are no legacy sites on this

Reply:
no legacy sites with this styling, this was newly added

Reply:
hmmmm the color is a good point though, it's not in the mock and it wasn't mentioned in the slack post
![image](https://user-images.githubusercontent.com/23005393/84689954-c1dbd000-af0f-11ea-8f9d-d9e3287eeea5.png)
so I think we can just remove it and life will be much simpler